### PR TITLE
[Documentation]Updated links that were redirecting to 404

### PR DESCRIPTION
**Pull request by benbd5**
| Q               | A                                                            |
|-----------------|--------------------------------------------------------------|
| Branch?         | 1.12                    |
| Bug fix?        | no                                                      |
| New feature?    | no                                                      |
| BC breaks?      | no                                                       |
| Deprecations?   | no |
| Related tickets |                      |
| License         | MIT                                                          |

Updated links that were redirecting to 404 like : https://docs.sylius.com/book/orders/payments

### CODE REVIEW

File: .zshrc
Comment (diimpp) [inline .zshrc:1]: Please remove this file

**Review by diimpp — CHANGES_REQUESTED** [review]
Hi @benbd5, please remove `.zshrc` file and changes should be good to be merged :+1:

File: .zshrc
Comment (benbd5) [reply .zshrc:1]: It's done

**Review by diimpp — APPROVED** [review]

**Review by NoResponseMate — APPROVED** [review]

**Review by GSadee — APPROVED** [review]

**Comment by GSadee** [comment]
Thank you, Benjamin! :tada:

**Pull request merged**